File: clients/go/model/debug_data.go
package model

import "io"

// TODO - Document
type DebugData struct {
}

func NewDebugData() DebugData {
    return DebugData {
    }
}

// Read DebugData from reader
func ReadDebugData(reader io.Reader) DebugData {
    return DebugData {
    }
}

// Write DebugData to writer
func (debugData DebugData) Write(writer io.Writer) {
}

// Get string representation of DebugData
func (debugData DebugData) String() string {
    stringResult := "{ "
    stringResult += " }"
    return stringResult
}

File: clients/csharp/Model/DebugData.cs
namespace SpbAiChamp.Model
{
    /// <summary>
    /// TODO - Document
    /// </summary>
    public struct DebugData
    {
    
        /// <summary> Read DebugData from reader </summary>
        public static DebugData ReadFrom(System.IO.BinaryReader reader)
        {
            var result = new DebugData();
            return result;
        }
    
        /// <summary> Write DebugData to writer </summary>
        public void WriteTo(System.IO.BinaryWriter writer)
        {
        }
    
        /// <summary> Get string representation of DebugData </summary>
        public override string ToString() {
            string stringResult = "DebugData { ";
            stringResult += " }";
            return stringResult;
        }
    }
}